embedded-graphics/src/style/styled.rs

```diff
@@ -42,15 +43,8 @@ impl<T, S> Dimensions for Styled<T, S>
 where
     T: Dimensions,
 {
-    fn top_left(&self) -> Point {
-        self.primitive.top_left()
-    }
-
-    fn bottom_right(&self) -> Point {
-        self.primitive.bottom_right()
-    }
-
-    fn size(&self) -> Size {
-        self.primitive.size()
+    //FIXME: The bounding box for a styled primitive should use the stroke width and alignment.
```

Comment:
If you're not going to fix this soon, can you create an issue so it doesn't get forgotten about please?

Reply:
These should be fixed soon, but I've started a tracking issue for these changes.